ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -67,3 +67,11 @@ for (const chain in FIXTURES) {
   }
   await writeFile(chainModuleFile, chainModule);
 }
+
+async function prepareReceiptProofFixture(client: AlchemyClient, block: Block, txHash: Hash, modulePath: string) {
+  const txIdx = block.transactions.indexOf(txHash);
```

Comment:
I think it will be more readable if we use .find instead of .indexOf. Then we don't need -1 const

Reply:
I'm not sure if it's gonna be more readable, as .find is a more complex function

Reply:
Agree. It will be if we fetch block with transactions, but it requires refactors in other places and we don't need it for receipts.